### DISCUSSION TITLE
Collapsing breakpoints

### DISCUSSION THREAD

**Discussion by happymacarts** [rappasoft/laravel-livewire-tables#1228] [Feature Requests]
Wanted to start this thread for possible addition of additional collapses for medium, large and other breakpoints to see if there is a need for it

**Comment by lrljoe** [comment]
FYI - I've created an Issue ([Feature Request]) for this to allow us to track it in the project

**Comment by lrljoe** [comment]
Are you using tailwind 2 or 3 or bootstrap 4 or 5 by the way. 

 Ultimately any solution for the package will have to support them all, but always good to know for any early testing opportunities.
